OKTA-648462 Minor Angular SPA redirect guide updates (#4418)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md

The `OktaAuthGuard` can protect a route to a single component as well with `canActivate` property.

1. Add the `canActivate` property to the `routes` array in `app-routing.module.ts`:

   ```ts
   { path: 'profile', component: ProfileComponent, canActivate: [OktaAuthGuard] }
   ```

> The single `/profile` route is protected directly, and on launch goes directly to the sign-in page (no need to click the sign-in button).

Reviewer: I'm not sure I understand this. Would you mind clarifying for me?

Reviewer: Oh gotcha. I see what you mean. Would something like this work?

> The single `/profile` route is protected directly, and on navigation will redirect you to the sign-in page if you aren't already authenticated.

Author: Perfect. Thanks!

> The single `/profile` route is protected directly, and when navigating to the page it prompts you to sign in first before routing you to the `ProfileComponent`.

Reviewer 2: ```suggestion
The single `/profile` route is protected directly. When you navigate to the page, you must sign in before being routed to the `ProfileComponent`.
```

Reviewer: Hey @isaacdurham-okta! How would you recommend wording this if the user is already authenticated? In that case, the user isn't prompted to sign in before navigating the route.

Author: I think we're trying to highlight that you can access this route if authenticated and, if not, you're prompted to sign in. How about:

_The single `/profile` route is protected directly. If you're not authenticated and navigate to this page, you must sign in before being routed to the `ProfileComponent`._ 

Is that too clunky? @alisaduncan @isaacdurham-okta

Reviewer: LGTM!

Reviewer 2: LGTM. Sorry, I missed your comments yesterday!
